cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -858,6 +860,27 @@ func main() {
 		os.Exit(0)
 	}
 
+	if *debugListIndexed {
+		indexed := listIndexed(s.IndexDir)
+		for _, r := range indexed {
+			fmt.Println(r)
+		}
+		os.Exit(0)
+	}
+
+	if *debugFind != "" {
+		args := indexArgs{
+			IndexOptions: IndexOptions{
+				Name: *debugFind,
+			},
+			IndexDir: *index,
+		}
+		bo := args.BuildOptions()
+		shards := bo.FindAllShards()
+		fmt.Printf("shards=%+v\n", shards)
```

Comment:
a shard per line might be easier to parse? Same as you did in the listindexed debug call.